v2/api/devices/customizations/iot_hub_extensions_test.go

```diff
@@ -32,6 +32,8 @@ func Test_SecretsSpecified_AllSecretsSpecifiedAllSecretsReturned(t *testing.T) {
 	}
 	secretNames := secretsSpecified(obj)
 	expectedTags := reflecthelpers.GetJSONTags(reflect.TypeOf(devices.IotHubOperatorSecrets{}))
+	// We expect every property in the secrets struct to be considered a secret except for the $propertyBag one
```

Comment:
Seems like this shouldn't be showing up in the diff? I thought I added this?

Reply:
Something is very off with this PR - the changes should be only for `batch`. I'll fix this up and force push.